.github/dependabot.yml

```diff
@@ -109,3 +109,24 @@ updates:
     directory: /plugin/storage/cassandra
     schedule:
       interval: daily
+  - package-ecosystem: docker
+    directory: /docker-compose/elasticsearch
+    schedule:
+      interval: daily
+    allow:
+      - dependency-name: "docker.elastic.co/elasticsearch/elasticsearch"
+        update-types: ["version-update:semver-minor"]
```

Comment:
neat, I didn't know this was possible, and it addresses one of my concerns in the ticket (to avoid it auto-upgrade 7x -> 8x)